Regression/WarpX-tests.ini

```diff
@@ -2176,7 +2176,7 @@ useOMP = 1
 numthreads = 1
 compileTest = 0
 doVis = 0
-analysisRoutine = Examples/analysis_default_regression.py
+analysisRoutine = Examples/analysis_default_openpmd_regression.py
```

Comment:
Please remove `amr.n_cell=384 512 max_step=100` above and add it to the inputs file instead. Comment the recommended values as before.